webui/react/src/stores/workspaces.tsx

```diff
@@ -17,7 +17,10 @@ import { Loadable, Loaded, NotLoaded } from 'utils/loadable';
 
 import PollingStore from './polling';
 
+const FETCH_EXPIRATION = 3600 * 1000;
```

Comment:
an hour seems like a long polling interval to me, but I'm not sure what best practices are for something like this